EnglishFilter/sections/foreign.txt

```diff
@@ -6308,6 +6308,8 @@ roweroweporady.pl,tulodz.pl#@#.ad_container
 !+ PLATFORM(ios, ext_android_cb)
 @@||lib.wtg-ads.com/prebid/prebid_*.js$domain=tvn.pl
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/116892
+||rynekinfrastruktury.pl/js/_*.js
```

Comment:
Better do it via Regex. Something like that
```
/rynekinfrastruktury\.pl\/js\/_[a-zA-Z]{17}\.js/$domain=rynekinfrastruktury.pl
```
@AdamWr could you also take a look?

Reply:
```
/\/js\/_[a-zA-Z]{17}\.js$/$domain=rynek-kolejowy.pl|rynek-lotniczy.pl|transport-publiczny.pl|rynekinfrastruktury.pl
```
This works ok on my end, but Safari do not support it.
So, we need your rule for content blockers.
```
rynek-kolejowy.pl,rynek-lotniczy.pl,transport-publiczny.pl,rynekinfrastruktury.pl#@#.adBanner
```

Reply:
same here https://github.com/AdguardTeam/AdguardFilters/issues/116902

Reply:
it seems ads or/and analytics
![image](https://user-images.githubusercontent.com/8361299/165933562-ed1c0710-bb3b-4fa6-be56-8cbbe0b86324.png)

Reply:
`portalcms.bm5.pl/Stat/statystyki.aspx` probably can be blocked, but I'm not sure about `portalcms.bm5.pl/Ajax/GetIloscWyswietlen.aspx`, because it will cause that widget with number of views will be blocked (market on a screenshot below will be not visible).
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/165936988-2409ef4a-cdeb-4cee-85d0-5873e514d2a4.png)

</details>

`portalcms.bm5.pl/Ajax/getReklamaRandom.aspx` can be also blocked, though I cannot reproduce this request with AdGuard enabled.

Reply:
So, I think that these rules can be added to `foreign.txt`:
```
||portalcms.bm5.pl/Ajax/getReklamaRandom.aspx
/\/js\/_[a-zA-Z]{17}\.js$/$domain=rynek-kolejowy.pl|rynek-lotniczy.pl|transport-publiczny.pl|rynekinfrastruktury.pl
rynek-kolejowy.pl,rynek-lotniczy.pl,transport-publiczny.pl,rynekinfrastruktury.pl#@#.adBanner
```
and this to `Tracking Protection filter`:
```
||portalcms.bm5.pl/Stat/statystyki.aspx
```